pkg/util/utils.go

```diff
@@ -600,3 +600,12 @@ func HomeDir() (string, error) {
 	}
 	return home, nil
 }
+
+func Tmpdir() string {
+	tmpdir := os.Getenv("TMPDIR")
+	if tmpdir == "" {
+		tmpdir = "/var/tmp"
+	}
+
+	return tmpdir
+}
```

Comment:
Does this cause the `--tmpdir` global option to be ignored?

Reply:
No.

Reply:
There were already being ignored.

Man page for podman documents this.

**--tmpdir**

Path to the tmp directory, for libpod runtime content.

NOTE --tmpdir is not used for the temporary storage of downloaded images.  Use the environment variable `TMPDIR` to change the temporary storage location of downloaded container images. Podman defaults to use `/var/tmp`.

Reply:
My bad, I'd pulled up the man page and in my rush neglected to read the note.  Ugh.
LGTM

Reply:
Hi! Thanks for making this change. I was surprised to see my /var filling up when I thought I had locked it down to only daemons.

It's unfortunate this is inconsistent with `mktemp` which defaults to `/tmp` when TMPDIR is unset. I don't suppose you would consider making `/tmp` the default?

Reply:
@kouso unless my memory banks have fogged over, I believe we originally had /tmp as the default several versions ago, but ran into a lot of issues with container builds not working as there wasn't enough space.

Reply:
Interesting! Well, I suppose it depends on your setup.

I can just set up a /etc/profile.d/ file to get `mktemp` and `podman` to agree, so it's not a big deal. Thanks for taking the time to fill me in.